webrender/src/prim_store.rs

```diff
@@ -58,15 +58,38 @@ pub struct PrimitiveCacheInfo {
     pub key: PrimitiveCacheKey,
 }
 
+#[derive(Debug)]
+pub enum PrimitiveClipInfo {
```

Comment:
Is the idea here that it gets resolved from Complex / Region to GpuAddress? Would it make more sense to split into a "source" and "resolved" field perhaps?